core/new-gui/src/app/dashboard/user/component/share-access/share-access.component.ts

```diff
@@ -66,7 +68,13 @@ export class ShareAccessComponent implements OnInit {
           this.validateForm.get("accessLevel")?.value
         )
         .pipe(untilDestroyed(this))
-        .subscribe(() => this.ngOnInit());
+        .subscribe(() => {
+          this.ngOnInit();
+          this.gmailService.sendEmail(
+            this.owner + " shared a " + this.type + " with you",
```

Comment:
The message is too abrupt. Try making it more informative, e.g., at least mention "Texera" in the email, otherwise a user would be confused.

Reply:
Extened.